brainscore_vision/benchmarks/imagenet/benchmark.py

```diff
@@ -22,7 +22,7 @@ def __init__(self):
         self._stimulus_set = stimulus_set
         self._similarity_metric = load_metric('accuracy')
         ceiling = Score(1)
-        super(Imagenet2012, self).__init__(identifier='fei-fei.Deng2009-top1', version=1,
+        super(Imagenet2012, self).__init__(identifier='Deng2009-top1', version=1,
```

Comment:
Not necessary, but maybe you could tackle this in the same PR: we changed the names of engineering/ML benchmarks to the more canonical community name, e.g. `ImageNet-top1` for this case: https://github.com/brain-score/vision/blob/65e2c4608a00bff6cd791f920498d19eab7ac33a/brainscore_vision/benchmarks/imagenet/__init__.py#L4
It would make sense to update the identifier outside the registry too (will require a database update which I can take care of)